eng/submit-source-build-release-pr.sh

```diff
@@ -135,7 +150,43 @@ cat "$global_json_path" \
     | tee "$global_json_path.new"
 mv "$global_json_path.new" "$global_json_path"
 
-sed -i "s#<PrivateSourceBuiltArtifactsPackageVersion>.*</PrivateSourceBuiltArtifactsPackageVersion>#<PrivateSourceBuiltArtifactsPackageVersion>$sdk_version</PrivateSourceBuiltArtifactsPackageVersion>#" $versions_props_path
+# Function to modify the URL in an XML file
+# Arguments:
+#   1. XML file path
+#   2. Element name
+#   3. Replacement value
+function modify_url_in_xml() {
+  local xml_file="$1"
+  local element_name="$2"
+  local replacement_value="$3"
+
+  # Fetch the URL inside the element
+  local url=$(grep -oP "<$element_name>\K.*?(?=<\/$element_name>)" "$xml_file")
```

Comment:
Works with generic value, not just URLs.

Reply:
Have you checked what happens when the element is not found?

Reply:
Done, thank you :)